protobuf/mayastor.proto

```diff
@@ -414,6 +415,7 @@ message PublishNexusRequest {
   string uuid = 1; // uuid of the nexus which to create device for
   string key = 2; // encryption key
   ShareProtocolNexus share = 3;  // protocol used for the front end.
+  repeated string allowed_hosts = 4; // host (nqn's) which are allowed to connect to the target.
```

Comment:
This is for controlling initiator's access to nexus right?

Reply:
Yes

Reply:
Thanks